scripts/delete.rb

```diff
@@ -0,0 +1,2 @@
+# Kill off, er, I mean delete one of the people. Look them up by their title.
+Person.where('title = :title', title: 'Hand of the King and Master of Coin').destroy
```

Comment:
`.where` returns an array of results, even if there is only one match. You can either use `.find_by` to return the first match and destroy it, or use `.destroy_all` to destroy everything in the array